resources/user-guides/attach-data.md

```diff
@@ -251,24 +256,24 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### Attaching additional Target parameters to Target location-clicked events
+### Attaching additional data to Target location-clicked events<a name="Target-Attach-Data-Location-Clicked"></a>
 
 The following sample shows how to add additional mbox and profile parameters in all outgoing `locationClicked` Target network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#target-create-rule-clicked)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#target-select-an-event-clicked)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#target-define-the-action-clicked)
-4. [Save and rebuild the property](attach-data.md#target-save-the-rule-and-rebuild-your-property-clicked)
+1. [Create a new **Rule**](#target-create-rule-clicked)
+2. [Select the **Event** you want to trigger the rule](#target-select-an-event-clicked)
+3. [Select the **Action** to attach data and define your payload](#target-define-the-action-clicked)
+4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-clicked)
```

Comment:
Please add a period to the end of each step, like this:
"4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-clicked)."